metadata.rb

```diff
@@ -8,6 +8,7 @@
 
 depends "apt"
 depends "chef-client"
+depends "compat_resource"
```

Comment:
This cookbook is deprecated

https://supermarket.chef.io/cookbooks/compat_resource